core/sile.lua

```diff
@@ -112,10 +112,11 @@ end
 SILE.init = function ()
   -- Set by def
   if not SILE.backend then
-    if pcall(require, "justenoughharfbuzz") then
+    local status, result = pcall(require, "justenoughharfbuzz")
+    if status then
```

Comment:
Is this a typo or does libtexpdf backend really require justenoughharfbuzz module?

Reply:
By default we require libtexpdf & harfbuzz pairing if not otherwise specified, and this looks on purpose ... make usre we have a working default shaper before we assume the default backend. But the code does look fishy, I'll give it a review it a bit...

Reply:
At least the error message can be improved if this is on purpose. It talks about missing libtexpdf, but it is harfbuzz that is missing.

Reply:
I believe the this bit of obtuse code has outlived its original purpose. I've simplified it and errors should make more sense now. We don't have a fallback for a different backend than the libtexpdf/harfbuzz combo right now. If/when we do configure flags should be able to change the default, not just a try/fail.